python/artm/model.py

```diff
@@ -250,21 +246,19 @@ def remove_dictionary(self, dictionary_name=None):
         else:
             raise IOError('dictionary_name is None')
 
-    def fit_offline(self, batch_vectorizer=None, num_collection_passes=1, decay_weight=0.0,
-                    apply_weight=1.0, reset_theta_scores=False):
+    def fit_offline(self, batch_vectorizer=None, num_collection_passes=10,
+                    num_document_passes=1, reuse_theta=True):
```

Comment:
If a user wants to "fit_offline" his model and he doesn't want to think about anything else, how many passes should be performed? For sure, it depends on the size of the collection, but fit_offline is supposed to be performed for small collections like kos or NIPS or MRRO. Then, 1 is not enough, while too many might take too long. So 10 =)